legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/to/DEPRECATED_PureGrammarComposerCore.java

```diff
@@ -881,9 +882,9 @@ public String visit(CLatestDate cLatestDate)
     }
 
     @Override
-    public String visit(CByteStream cByteStream)
+    public String visit(CByteArray cByteArray)
     {
-        return "byteStream(" + HelperValueSpecificationGrammarComposer.renderString(cByteStream.value, this) + ")";
+        return "byteArray(" + HelperValueSpecificationGrammarComposer.renderString(new String(cByteArray.value, StandardCharsets.UTF_8), this) + ")";
```

Comment:
Are we sure we want to render the bytes in this way? If the bytes originally came from a string, it will be fine; but if they didn't, it might well yield a problematic string (possibly including newlines and special characters and such). Maybe it would be better to encode them in base 64 or something.

Reply:
We essentially use this in context of service tests data so we can't have an encoding else it won't be possible for users to work with tests.
Regarding special characters like new-line our antlr parser only allow STRING token so new-line or anything special should already have been handled appropriately. 
Test : testBindingWithService()